cmd/builder/.goreleaser.yml

```diff
@@ -23,6 +24,80 @@ builds:
       - goos: windows
         goarch: arm64
     binary: ocb
+dockers:
+  - goos: linux
+    goarch: amd64
+    dockerfile: Dockerfile
+    image_templates:
+      - johnpeterson785/opentelemetry-collector-builder:{{ .Version }}-amd64
```

Comment:
Is this `johnpeterson785/opentelemetry-collector-builder:{{ .Version }}-amd64` used for testing? Should this be moved?

Reply:
thank you, good catch. We had `otel/opentelemetry-collector-builder` created on Dockerhub so I have updated these references